spec/datadog/core/workers/runtime_metrics_spec.rb

```diff
@@ -15,12 +15,12 @@
   after { worker.stop(true, 1) }
 
   shared_context 'short default flush interval' do
-    let(:default_flush_interval) { 0.1 }
-    before { stub_const('Datadog::Workers::RuntimeMetrics::DEFAULT_FLUSH_INTERVAL', default_flush_interval) }
+    let(:default_flush_interval) { 0.3 }
+    before { stub_const('Datadog::Core::Workers::RuntimeMetrics::DEFAULT_FLUSH_INTERVAL', default_flush_interval) }
   end
```

Comment:
Curious about this change -- was it the flakiness I predicted from https://github.com/DataDog/dd-trace-rb/pull/1858#discussion_r790831263 ? If so, I think the solution would be to use wait for as a mechanism for waiting a bit of extra time in case things are running slow, rather than impacting the full test suite by raising the time for the `sleep`.

Reply:
Reducing the interval to `0.1` in the test was too short; it could take longer than that to complete its second call. Point of the test is to wait an interval period and expect it did its job. There's a challenge in implementing your suggestion; see my response to that comment for details.